tasks/push-snapshot/tests/test-push-snapshot.yaml

```diff
@@ -86,5 +89,13 @@ spec:
                 cat $(workspaces.data.path)/mock_skopeo.txt
                 exit 1
               fi
+
+              test $(jq -r '.images[0].name' $(workspaces.data.path)/results/push-snapshot-results.json) == "comp"
```

Comment:
Nice. I was looking to see if at least one test tests this and sure enough, the last one does :)